src/components/DisbursementButtons/index.tsx

```diff
@@ -3,6 +3,7 @@ import { useNavigate } from "react-router-dom";
 import { Routes } from "@/constants/settings";
 import { DisbursementDraftAction, DisbursementStep } from "@/types";
 import "./styles.scss";
+import { ShowForRoles } from "@/components/ShowForRoles";
```

Comment:
Nit: types and relative imports (styles) should be the last. We should move this above types.